client/src/components/calendarDay.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { RegularDay } from '../styles/styles.jsx';
+import { AvailableDay } from '../styles/styles.jsx';
+import { EmptyDay } from '../styles/styles.jsx';
+import { Div } from '../styles/styles.jsx';
+
+class CalendarDay extends React.Component {
+  
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if( this.props.day && this.props.cost === undefined || this.props.cost === '' ){
+      return (
+        <RegularDay>
+          <Div>
```

Comment:
If you are using styled components, you don't need this div inside of here. The styled component is basically a div.